Sources/AmazonLocationiOSAuthSDK/Cognito/CognitoCredentials.swift

```diff
@@ -0,0 +1,48 @@
+import Foundation
+
+public struct CognitoCredentials: Codable {
+    public let identityPoolId: String
+    public let accessKeyId: String
+    public let secretKey: String
+    public let sessionToken: String?
+    public let expiration: Date?
+
+    public init(identityPoolId: String, accessKeyId: String, secretKey: String, sessionToken: String? = nil, expiration: Date? = nil) {
+        self.identityPoolId = identityPoolId
+        self.accessKeyId = accessKeyId
+        self.secretKey = secretKey
+        self.sessionToken = sessionToken
+        self.expiration = expiration
+    }
+    
+    
+    public static func encodeCognitoCredentials(credential: CognitoCredentials) -> String? {
+        let encoder = JSONEncoder()
+        encoder.dateEncodingStrategy = .iso8601
+        do {
+            let jsonData = try encoder.encode(credential)
+            let jsonString = String(data: jsonData, encoding: .utf8)
+            return jsonString
+        } catch {
+            print("Error encoding CognitoCredential to JSON: \(error)")
```

Comment:
Instead of `print` is there a logger we can use instead? (same below)

Reply:
Implemented the logger for all prints

Reply:
Thank you!